plugin-server/src/utils/db/hub.ts

```diff
@@ -161,7 +160,10 @@ export async function createHub(
 
     status.info('🤔', `Connecting to Kafka...`)
     const kafka = new Kafka({
-        clientId: `plugin-server-v${version}-${instanceId}`,
+        /* clientId does not need to be unique, and is used in Kafka logs and quota accounting.
+           os.hostname() returns the pod name in k8s and the container ID in compose stacks.
+           This allows us to quickly find what pod is consuming a given partition */
+        clientId: hostname(),
```

Comment:
What would happen if you ran two plugin-servers locally with different `PLUGIN_SERVER_MODE`s? Would this cause problems? Its theoretical atm so no need to block, but interested to know what the Kafka coordinator behaviour would we in this case.

Reply:
From a quick read of the docs it looks like this is mainly just for humans and better visibility into what is making a request to Kafka. Nice debugging trick

https://kafka.js.org/docs/configuration#client-id